kong/api/app.lua

```diff
@@ -146,8 +147,8 @@ for _, v in ipairs({"kong", "apis", "consumers", "plugins", "cache", "cluster" }
 end
 
 -- Loading plugins routes
-if configuration and configuration.plugins then
-  for _, v in ipairs(configuration.plugins) do
+if singletons.configuration and singletons.configuration.plugins then
```

Comment:
@thibaultCha Any idea why this is `configuration.plugins` while in kong.lua it's declared as `configuration.loaded_plugins`?